src/shared/generator.c

```diff
@@ -64,6 +64,7 @@ static int write_fsck_sysroot_service(const char *dir, const char *what) {
                 "Documentation=man:systemd-fsck-root.service(8)\n"
                 "Description=File System Check on %2$s\n"
                 "DefaultDependencies=no\n"
+                "ConditionPathExists=%2$s\n"
```

Comment:
Would it be worth it to change this into an `AssertPathExists` so at least it fails with a clear error on what is wrong?